docker-compose.override.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.4'
```

Comment:
Can we solve this with 1 compose?

Reply:
Can you be clearer? Are you talking about the override file?

Reply:
For the result we want to achieve we don't necessarily need to utilize the override file that docker supports, we can initialize the containers just with 1 compose

Reply:
Cool, I cleaned it up in the last commit.